src/Serializer/Normalizer/AggregateIdNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Serializer\Normalizer;
+
+use Attribute;
+use Cspray\Phinal\AllowInheritance;
+use Patchlevel\EventSourcing\Aggregate\AggregateRootId;
+use Patchlevel\Hydrator\Normalizer\InvalidArgument;
+use Patchlevel\Hydrator\Normalizer\Normalizer;
+
+use function is_string;
+
+#[Attribute(Attribute::TARGET_PROPERTY)]
+#[AllowInheritance('you can make specific normalizers for different classes')]
+class AggregateIdNormalizer implements Normalizer
```

Comment:
`AggregateIdNormalizer `  maybe to => `AggregateRootIdNormalizer` to match the `AggregateRootId` class name.